DICOM/src/Microsoft.Health.DeID.SharedLib/EncryptFunction.cs

```diff
@@ -167,7 +180,7 @@ public static byte[] EncryptContentWithRSA(byte[] plainBytes, byte[] publicKey,
                 byte[] encryptedData;
                 using (RSACryptoServiceProvider rSA = new RSACryptoServiceProvider())
                 {
-                    rSA.ImportRSAPublicKey(publicKey, out _);
+                    rSA.ImportRSAPublicKey(_publicKey, out _);
```

Comment:
>ImportRSAPublicKey [](http://example.com/codeflow?start=24&length=18)

Can you explain what this encryption is used for?

Reply:
Removed RSA for now.